_emails/tacoma.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Tacoma, Washington
+permalink: "/tacoma"
+name: Letter to Tacoma City Officials
+state: WA
+city: Tacoma
+layout: email
+recipients:
+- Chris.beale@cityoftacoma.org 
+- Victoria.woodwards@cityoftacoma.org 
+- nicole.nowman@cityoftacoma.org
+subject: Justice for the Manslaughter of Manuel Ellis
+body: "Hello Tacoma Officials,\n\n
+
+As a concerned citizen I am writing to you on behalf of the family of Manuel Ellis. I am demanding justice for the murder, by Tacoma law enforcement, of Manuel in Tacoma on March 3rd, 2020. Justice includes the release of compliant history of his murder, disciplinary record, toxicology reports of Manuel and the officers involved, release of autopsy and footage to the family of Manuel Ellis.\n\n
+
+Additionally, please use the power appointed by the people to you to fire and prosecute to officers Matthew Collins, Christopher Burbank, Masyih Ford, and Timothy Rankine involved in the manslaughter of Manuel Ellis. To further prosecute the injustice unpunished by police chief Donald Ramsdell, please terminate him immediately and replace him with an anti-racist and justice leader.\n\n
+
```

Comment:
This is really sad, but also great. Can we include a brief bit about the Tacoma police budget and changes that we are requesting?